sdks/python/container/py37/base_image_requirements.txt

```diff
@@ -66,7 +66,7 @@ google-cloud-profiler==3.1.0
 google-cloud-pubsub==2.13.7
 google-cloud-pubsublite==1.5.0
 google-cloud-recommendations-ai==0.7.1
-google-cloud-spanner==1.19.3
+google-cloud-spanner==3.22.0
```

Comment:
Tests fail with:
09:43:29 google-cloud-spanner 3.22.0 requires sqlparse, which is not installed.

these files need to be regenerated (see header at the top of the file).

I think it would be safer to set the upper bound >=3.0.0 in requirements file.